tfx/components/example_gen/write_split.py

```diff
@@ -48,22 +52,36 @@ def process(self, e: Union[tf.train.Example, tf.train.SequenceExample,
 
 @beam.ptransform_fn
 @beam.typehints.with_input_types(
-    Union[tf.train.Example, tf.train.SequenceExample, bytes])
+    Union[tf.train.Example, tf.train.SequenceExample, bytes, pa.Table])
 @beam.typehints.with_output_types(beam.pvalue.PDone)
 def WriteSplit(
     example_split: beam.pvalue.PCollection,
     output_split_path: str,
     output_format: str,
+    exec_properties: Dict[str, Any]
 ) -> beam.pvalue.PDone:
-  """Shuffles and writes output split as serialized records in TFRecord."""
+  """Shuffles and writes output split as serialized records in TFRecord or Parquet."""
```

Comment:
line <= 80 chars, so you need to format it like this:

"""
xxxxxxxxxxxxxxxxxxxxxx
xxxxxxxxxxxxxxxxxxxxxx
"""